backend/src/validators/units.ts

```diff
@@ -90,6 +90,7 @@ const createUnitSchema = [
   body("numBeds").exists().withMessage("is required").isNumeric().withMessage("must be a number"),
   body("numBaths").exists().withMessage("is required").isNumeric().withMessage("must be a number"),
   body("appliances").exists().withMessage("is required").isArray().withMessage("must be an array"),
+  body("utilities").exists().withMessage("is required").isArray().withMessage("must be an array"),
```

Comment:
I'm not sure if this is necessary, but if this section is required but the landlord doesn't provide any of the the utilities can they still submit the form? Should there be an N/A section also or are they just not allowed to submit the unit if they don't provide at least one of those utilities?

Reply:
Oh ok I see that you can actually submit without filling out the utilities section, but then should it be labeled with the required star?

Reply:
Yeah that's for sure an inconsistency. This issue also applies to other fields on the form, so I think we should address this in another task. Thanks for bringing it up, though!